sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/changefeed/epkversion/FullFidelityChangeFeedProcessorTest.java

```diff
@@ -1908,8 +1909,15 @@ void validateChangeFeedProcessing(
     void validateChangeFeedProcessorContext(ChangeFeedProcessorContext changeFeedProcessorContext) {
 
         String leaseToken = changeFeedProcessorContext.getLeaseToken();
-
         assertThat(leaseToken).isNotNull();
+
+        CosmosDiagnostics diagnostics = changeFeedProcessorContext.getDiagnostics();
```

Comment:
Similar tests shoudl be added to IncrementalChangeFeedProcessorTest in the epkversion?

Reply:
Oh - and this won't compile? CosmosDiagnostics vs. CosmosDiagnosticsCOntext?

Reply:
It compiles, the context is just null

Reply:
fixed based on offline conversation